Makefile

```diff
@@ -96,6 +96,7 @@ $(OUT)/storage_%.tz: $(shell find ligo -name '*.mligo')
 			; y_token_id = $(y_token_id)n \
 			; x_token_address = (\"$(x_token_address)\" : address) \
 			; y_token_address = (\"$(y_token_address)\" : address) \
+			; tick_spacing = (1n : nat) \
```

Comment:
This should be an argument of this target as well.

Reply:
And it should be mentioned in `docs/compilation.md`

Reply:
Right, I meant to update this before pushing the PR but forgot. Also update the `compilation.md` as well.